### ISSUE TITLE
blurEdge not fired on hoverNode

### ISSUE THREAD

**Issue by parthms** [almende/vis#1911]
Firstly, I wanted to tell you that you have done an excellent job with Vis.js. I absolutely love it.

I think I found a bug, let me know if I'm approaching the problem incorrectly. So here is my interaction options:

`interaction: {
     hover: true,
     hoverConnectedEdges: false,
     selectConnectedEdges: false
}`

Based on these settings, what I'm trying to do is separate the selection of an edge and a node. Meaning you can only select an edge or a node, not a group of them.

The issue at hand is that blurEdge is not fired on hoverNode. If I move the cursor straight from an edge to a node, then the hover styles are not removed from the edge.

**Comment by Tooa** [comment]
Can you please test with the latest visjs version again? We have tackled a few issues related to `blurEdge` and `hoverNode` in our latest release.

**Comment by bthieurmel** [comment]
Regarding this fiddle https://jsfiddle.net/4zt6a71m/1/, changing ressource ``vis.js`` 4.14 to 4.17, it's seems that the problem already exists : passing from edge to a node, ``blurEdge`` is not called.

Moreover, now, ``hoverEdge`` and ``blurEdge`` are called many times staying on same edge.

**Comment by Tooa** [comment]
Indeed you found a bug. Especially, the moving from an edge to a node might be a corner case, which has been not addressed in the code.

**Comment by bthieurmel** [comment]
Ok. Thanks. So waiting for fix and ready to test.

**Comment by dockstreet** [comment]
This fix would really help out, looking forward to it

**Comment by Tooa** [comment]
@bthieurmel @dockstreet I've created a pull request to resolve the issue here https://github.com/almende/vis/pull/2554. Can you please test whether it works for you two? There is quite a lot of logic going on in the part I've changed. So, make sure nothing else is broken.

**Comment by dockstreet** [comment]
short answer: 
I created a couple of plunkers. This fix seems to work, in regards to the issue found. 

long answer:
That said, I believe I may have uncovered another issue in the current development snapshot with IE11 and network. 

When I try to do anything with network in IE11, I get :

'Symbol' is undefined

This which does not occur in vis 4.17.0 with network .
This behavior does not occur in Edge/Chrome/Firefox with the Snapshot 4.18 version, only IE11.

I slightly modfied @bthieurmel's example and created a plunker by pulling from the develop branch in git, as I couldn't refer to the development version on cloudflare or local one with . Then I manually uploaded a minimized version here, as plunker wouldn't accept the whole vis.js file for some quota reason 

Here is my plunkr, I would have used jsbin, but blocked by proxy ; and not sure how to reference external files with jsfiddle. 

https://plnkr.co/edit/MX1agpoRYgYbgC23NAhl?p=preview

hope this helps & thanks soo much @Tooa & @bthieurmel!

Let me know if there is something I can try next

**Comment by Tooa** [comment]
@dockstreet Can you create a new issue for this problem? It helps to maintain the overall overview.

**Comment by dockstreet** [comment]
will do & thanks

**Comment by mojoaxel** [comment]
fixed by @Tooa with #2554
